contrib/terraform/openstack/group_vars/all.yml

```diff
@@ -105,21 +105,18 @@ kube_apiserver_insecure_port: 8080 # (http)
 # into appropriate IP addresses. It's highly advisable to run such DNS server,
 # as it greatly simplifies configuration of your applications - you can use
 # service names instead of magic environment variables.
-# You still must manually configure all your containers to use this DNS server,
-# Kubernetes won't do this for you (yet).
 
-# Do not install additional dnsmasq
-skip_dnsmasq: false
```

Comment:
Sorry but a hoped to see both docker dns and dnsmasq_kubedns applicable, which is:
* hosts' DNS config is not changed
* Pods DNS config points to the dnsmasq ds cluster ip